fdbserver/MoveKeys.actor.cpp

```diff
@@ -1446,7 +1453,7 @@ ACTOR static Future<Void> startMoveShards(Database occ,
 						}
 
 						// Update dest servers for this range to be equal to servers
-						krmSetPreviouslyEmptyRange(&tr,
+						krmSetPreviouslyEmptyRange(&(tr.getTransaction()),
```

Comment:
nit: will `&tr` be sufficient?

Reply:
Yes. Was copied from startMoveKeys. Let me fix.